tests/x11/libreoffice/libreoffice_pyuno_bridge_no_evolution_dep.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Case 1503978 - LibreOffice: pyuno bridge
+# Maintainer: Grace Wang <gwang@suse.com>
+# Tags: poo#34141
+
+use base "x11test";
+use strict;
+use testapi;
+
+sub run {
+    # Open LibreOffice
+    x11_start_program('oowriter');
+    assert_screen 'test-ooffice-1';
```

Comment:
That should not be necessary as the x11_start_program call already checked the program after startup

Reply:
Thanks for pointing this out. I've read the x11_start_program carefully and knew why. 
Updated.